src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -1247,7 +1247,7 @@ export default class SourceFileLinter {
 	 * from there. Directly finding the type of the property is not possible from here as it's
 	 * missing some context.
 	*/
-	#isPropertyBinding(node: ts.NewExpression | ts.CallExpression, propName: string[] | undefined) {
+	#isPropertyBinding(node: ts.NewExpression | ts.CallExpression, propName: string[]) {
```

Comment:
[minor] Optional chaining operator "?" after propName in line 1263 is now also obsolete...

Reply:
Thanks!